SECURITY.md

```diff
@@ -0,0 +1,68 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+The OPI security policy aims to efficiently handle security vulnerabilities
+while minimizing public disclosure until a fix is available.
+
+Reporters are requested to submit security issues privately via the
+[GitHub advisory](https://github.com/opiproject/opi/security/advisories/new)
+system.
```

Comment:
I read through the git security advisory mechanisms and this seems like a secure way to have a security researcher report an issue and to keep it private. Am I right that only the security researcher reporting the issue and the repo/org admins are able to see the issue, until an admin adds a "contributor" to the repo?

Reply:
This is the exact point of having the *private* reporting enabled. By this we want to minimize the public advertisement until a fix is ready. Otherwise you would need to ask the reporter to contact the maintainers or some dedicated security group but through advisories GH attempts to standardize the processes and presents the reporter with a form to help collect the data. With standard security advisories (https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/creating-a-repository-security-advisory) only people with proper rights on the repo can do that. With private advisories anyone (security researcher) can report.